projectLive/R/mod_study_summary.R

```diff
@@ -148,16 +148,15 @@ mod_study_summary_server <- function(
       "study_table"
     ) 
     
-    merged_dataset() %>% 
+    merged_table() %>% 
       dplyr::select_at(
         unlist(c(param_list$group_columns, param_list$count_columns))
       ) %>% 
       dplyr::group_by_at(unlist(param_list$group_columns))%>% 
       dplyr::summarise_at(unlist(param_list$count_columns), dplyr::n_distinct) %>% 
```

Comment:
The Participating studies table is showing count 1 in all columns for studies that don't have uploaded data. eg: https://www.synapse.org/#!Synapse:syn4939894/wiki/235903

Reply:
Yep, dplyr:n_distinct counts NA values by default. This has been fixed.